api-client/src/test/java/com/desk/java/apiclient/DeskClientTest.java

```diff
@@ -26,14 +26,21 @@
 
 package com.desk.java.apiclient;
 
+import com.desk.java.apiclient.util.DeskClientUtils;
```

Comment:
The name looks correct. It's testing the DeskClient. The `DeskClientUtils` class was added to handle the url signing stuff.